internal/character/natasha/skill.go

```diff
@@ -0,0 +1,68 @@
+package natasha
+
+import (
+	"github.com/simimpact/srsim/pkg/engine/info"
+	"github.com/simimpact/srsim/pkg/engine/modifier"
+	"github.com/simimpact/srsim/pkg/key"
+	"github.com/simimpact/srsim/pkg/model"
+)
+
+const (
+	Skill key.Modifier = "natasha-skill"
+)
+
+func init() {
+
+	//Nat HOT from skill
+	modifier.Register(
+		Skill,
+		modifier.Config{
+			Duration:          2,
+			Stacking:          modifier.Unique,
```

Comment:
Stacking behavior is `ReplaceBySource` (want new applications to reset the duration/timer)